angular.json

```diff
@@ -1,6 +1,13 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
   "version": 1,
+  "cli": {
+    "analytics": false,
+    "cache": {
+      "enabled": false,
+      "environment": "ci"
+    }
```

Comment:
If I follow the documentation here https://angular.io/guide/workspace-config#cache-options this deactivates the cache and sets the cache active for the CI environment.

Reply:
Yes. You're right.

Reply:
I fix this.